modules/gdnative/gdnative.cpp

```diff
@@ -243,7 +243,7 @@ void GDNativeLibrary::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("set_symbol_prefix", "symbol_prefix"), &GDNativeLibrary::set_symbol_prefix);
 	ClassDB::bind_method(D_METHOD("set_reloadable", "reloadable"), &GDNativeLibrary::set_reloadable);
 
-	ADD_PROPERTY(PropertyInfo(Variant::OBJECT, "config_file", PROPERTY_HINT_RESOURCE_TYPE, "ConfigFile"), "set_config_file", "get_config_file");
+	ADD_PROPERTY(PropertyInfo(Variant::OBJECT, "config_file", PROPERTY_HINT_RESOURCE_TYPE, "ConfigFile", PROPERTY_USAGE_INTERNAL), "set_config_file", "get_config_file");
```

Comment:
What exactly does `PROPERTY_USAGE_INTERNAL` do? This property should be assignable via script for example but not shown in the inspector or saved inside the `.tres` . Does this do that?

Reply:
@karroffel you are right, `INTERNAL` is not the right one, it hides the property from the docs (leaves setters and getters).
The right value seems to be `0` (and it's used like that around the engine).
Maybe we should have a `PROPERTY_USAGE_NONE = 0` in `PropertyUsageFlags`?
@reduz